src/Testcontainers.Pulsar/PulsarBuilder.cs

```diff
@@ -4,7 +4,7 @@ namespace Testcontainers.Pulsar;
 [PublicAPI]
 public sealed class PulsarBuilder : ContainerBuilder<PulsarBuilder, PulsarContainer, PulsarConfiguration>
 {
-    public const string PulsarImage = "apachepulsar/pulsar:3.2.3";
+    public const string PulsarImage = "apachepulsar/pulsar:3.3.1";
```

Comment:
Typically, we avoid updating the image version of modules to prevent developers from encountering potential breaking. However, in this case, I think we can make an exception.

Reply:
If you would like to avoid updating the image version so often. How do you feel about setting the target version to a LTS version [release-policy](pulsar.apache.org/contribute/release-policy/) ?

Reply:
Yes, LTS sounds good. Since there doesn't seem to be a 3.0 tag, should we go with the latest patch, 3.0.6?

Reply:
Yes 👍 !